apis/elbv2/v1beta1/ingressclassparams_types.go

```diff
@@ -100,9 +133,14 @@ type IngressClassParamsSpec struct {
 	Scheme *LoadBalancerScheme `json:"scheme,omitempty"`
 
 	// InboundCIDRs specifies the CIDRs that are allowed to access the Ingresses that belong to IngressClass with this IngressClassParams.
+	// If this and `securityGroups` are both specified, `securityGroups.managed` must be `true`.
```

Comment:
nit: should it be `securityGroups.ManageInbound` must be `true`? Can we also call this out in ingress_class.md and also helm chart definition?